build/wix/mixxx.wxs

```diff
@@ -0,0 +1,429 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+ 
+  <!--
+    Need some Wix resources ? See 
+    https://helgeklein.com/blog/2014/09/real-world-example-wix-msi-application-installer/ 
+    https://www.firegiant.com/wix/tutorial/getting-started/
+    https://msdn.microsoft.com/en-us/library/aa368269%28VS.85%29.aspx
+    http://weblogs.sqlteam.com/mladenp/archive/2010/02/23/WiX-3-Tutorial-Generating-filedirectory-fragments-with-Heat.exe.aspx
+    https://bitbucket.org/scons/scons/wiki/WiX_Tool
+    http://blogs.technet.com/b/alexshev/archive/2008/02/10/from-msi-to-wix.aspx
+    http://stackoverflow.com/questions/15305718/removing-unwanted-nodes-from-wxs-file-by-applying-xslt-tranfsormation/15306740#15306740
+  -->
+ 
+  <!--
+    ====================================================================================
+    Defines & Variables
+  -->
+  
+  <!-- Full version number to display -->
+  <?define VersionNumber="!(bind.FileVersion.mixxxExe)" ?>
+ 
+  <!--
+  Upgrade code HAS to be the same for all updates.
+  Once you've chosen it don't change it.
+  !!! DO NOT REUSE IN ANY OTHER PROJECT !!!
+  -->
+  <?define UpgradeCode="921DC99C-4DCF-478D-B950-50685CB9E6BE" ?>
+ 
+  <!-- The URL for add/remove programs -->
+  <?define InfoURL="http://mixxx.org/" ?>
+   
+  <!-- 32-bit / 64-bit variables -->
+  <?if $(var.Platform) = x64 ?>
+    <?define Win64 = "yes" ?>
+    <?define arch = "64" ?>
+    <?define notarch = "32" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+    <?define baseBuildDir = "..\.." ?>
+  <?else ?>
+    <?define Win64 = "no" ?>
+    <?define arch = "32" ?>
+    <?define notarch = "64" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFilesFolder" ?>
+    <?define baseBuildDir = "..\.." ?>
+  <?endif ?>
+ 
+  <?define SourceDir="$(var.baseBuildDir)\dist$(var.arch)"?>
+  
+  <!--
+    ====================================================================================
+    Package start
+  -->
+
+  <!-- The upgrade code must never change as long as the product lives! -->
+  <!-- Product IDs must be autogenerated (*) or else major upgrades will not work -->
+  <Product Id="*"
+           Name="!(loc.ApplicationName)"
+           Language="!(loc.Language)"
+           Version="$(var.VersionNumber)"
+           Manufacturer="!(loc.ManufacturerFullName)"
+           UpgradeCode="$(var.UpgradeCode)" >
+   
+    <!-- Package IDs are valid for a single package version only - they are autogenerated by WiX -->
+    <!-- Let's require Windows Installer 4.0 (included in Vista) -->
+    <!-- And ALWAYS install per machine!!! -->
+    <Package Id="*"
+             Keywords="Installer"
+             InstallerVersion="400"
+             Compressed="yes"
+             InstallScope="perMachine"
```

Comment:
Hummm, I need to think a bit more about this.
Currently, I use system-wide registry entries (like uninstall entries) which need perMachine and admin privileges.

I think our current installer is wrong because is tries to write to system-wide registry even when installing per-user. I need to check that and let you know.